### PR TITLE
identity: init at 0.5.0

### PR DESCRIPTION
**Pull request by paveloom**
###### Description of changes

https://gitlab.gnome.org/YaLTeR/identity

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 223048` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>identity</li>
  </ul>
</details>

**Review by lucasew — APPROVED** [review]

**Review by lucasew — CHANGES_REQUESTED** [review]
You can automate the updates of the package with `nix-update-script` so r-ryantm will send the bump PR for you.

Example: https://github.com/NixOS/nixpkgs/blob/1a173c89426ab705aac55872740362bc1a0b6cfd/pkgs/games/cbonsai/default.nix#L20

**Comment by paveloom** [comment]
> You can automate the updates of the package with `nix-update-script` so r-ryantm will send the bump PR for you.

That's pretty neat, thanks!

**Review by lucasew — APPROVED** [review]

File: pkgs/applications/graphics/identity/default.nix [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
+    description = "A program for comparing multiple versions of an image or video";
+    maintainers = with maintainers; [ paveloom ];
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+  };
```
Comment (superherointj) [inline pkgs/applications/graphics/identity/default.nix:68]:
```suggestion
  meta = {
    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
    description = "A program for comparing multiple versions of an image or video";
    maintainers = with lib.maintainers; [ paveloom ];
    license = lib.licenses.gpl3Plus;
    platforms = lib.platforms.linux;
  };
```

Reasoning:
- https://github.com/atorres1985-contrib/nixpkgs/commit/451dfa6d84586b5cb83e6a7254689dff66b4523a

File: pkgs/applications/graphics/identity/default.nix [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "identity";
+  version = "0.5.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "YaLTeR";
+    repo = pname;
```
Comment (AndersonTorres) [inline pkgs/applications/graphics/identity/default.nix:25]:
```suggestion
    repo = "identity";
```

File: pkgs/applications/graphics/identity/default.nix [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    rev = "v${version}";
+    sha256 = "0l2v07pvcqi6mq8xw7n88jg0w03lj7cld5vm0r08i7mhrmavc4k4";
```
Comment (AndersonTorres) [inline pkgs/applications/graphics/identity/default.nix:27]: Use SRI hash.

File: pkgs/applications/graphics/identity/default.nix [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+, desktop-file-utils
+, gst_all_1
+, gtk4
+, libadwaita
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook4
+, nix-update-script
+}:
```
Comment (superherointj) [inline pkgs/applications/graphics/identity/default.nix:16]: IMO arguments are better alphabetically ordered.

File: pkgs/applications/graphics/identity/default.nix [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  ]);
+
+  buildInputs = [
+    gtk4
+    libadwaita
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-libav
```
Comment (superherointj) [inline pkgs/applications/graphics/identity/default.nix:57]: Order alphabetically.

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 223048` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>identity</li>
  </ul>
</details>

**Comment by paveloom** [comment]
@ofborg eval

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 223048` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>identity</li>
  </ul>
</details>

File: pkgs/applications/graphics/identity/default.nix [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    gst_all_1.gstreamer
+    gtk4
+    libadwaita
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
+    description = "A program for comparing multiple versions of an image or video";
```
Comment (superherointj) [inline pkgs/applications/graphics/identity/default.nix:64]:
```suggestion
    description = "A program for comparing multiple versions of an image or video";
    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
```

Comment (superherointj) [reply pkgs/applications/graphics/identity/default.nix:64]: `description` should be first.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 223048` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>identity</li>
  </ul>
</details>

**Review by superherointj — APPROVED** [review]

**Pull request merged**